src/redux/reducers/usersReducer.js

```diff
@@ -0,0 +1,81 @@
+import { FOLLOW, UNFOLLOW, SET_USERS } from '../actions/actionTypes';
+import { AVATAR_DEFAULT } from '../../utils/url-utils';
```

Comment:
Consider moving this import.
I already asked your to reorganize the order of imports several times.

Reply:
/fxd